.circleci/config.yml

```diff
@@ -1396,25 +1396,25 @@ workflows:
       #       - op-supervisor-docker-build
       #       - proofs-tools-docker-build
       #       - go-test-all
-      # - docker-build:
-      #     name: <<matrix.docker_name>>-docker-build
-      #     docker_tags: <<pipeline.git.revision>>,<<pipeline.git.branch>>
-      #     save_image_tag: <<pipeline.git.revision>>
-      #     matrix:
-      #       parameters:
-      #         docker_name:
-      #           - op-node
-      #           - op-batcher
-      #           - op-program
-      #           - op-proposer
-      #           - op-challenger
-      #           - proofs-tools
-      #           - op-dispute-mon
-      #           - op-conductor
-      #           - da-server
-      #           - op-supervisor
-      #           - cannon
-      # - cannon-prestate
+      - docker-build:
+          name: <<matrix.docker_name>>-docker-build
```

Comment:
Out of curiosity, where are these values populated?

Reply:
Line 1405 pass this parameter